site/docs/plugins/auto-retry.md

```diff
@@ -60,8 +60,8 @@ Similarly, if the request ever fails with `retry_after` larger than what is spec
 
 ```ts
 autoRetry({
-  maxRetryAttempts: 1, // only repeat requests once
-  maxDelaySeconds: 5, // fail immediately if we must wait 6+ seconds
+  maxRetryAttempts: 1, // repeat requests only once
+  maxDelaySeconds: 5, // fail immediately if we have to wait >5 seconds
```

Comment:
And then also switch the phrasing here to match:
```suggestion
  maxDelaySeconds: 5, // if we have to wait >5 seconds, fail immediately
```

Reply:
I think it's easier to understand if the "if" is not in the beginning.

Reply:
I think it's easier to understand if the "if" is not in the beginning.